instance/models/openedx_appserver.py

```diff
@@ -21,6 +21,7 @@
 """
 import yaml
 
+import requests
```

Comment:
nit: what do you think about moving this requests import next to the yaml import and leaving a space before the django imports? It think it makes imports more clear, and easier to read.

Reply:
:+1: we should probably use `isort` if we aren't already, or something equivalent.